packages/async-rewriter/src/async-writer-babel.ts

```diff
@@ -748,4 +749,13 @@ export default class AsyncWriter {
   compile(code): string {
     return this.getTransform(code).code;
   }
+
+  /**
+   * Returns translated code.
+   * @param {string} code - string to compile.
+   */
+  process(code): string {
```

Comment:
Could you throw a test or two in here just to make sure it's doing what it's supposed to be doing?

Reply:
Just did! Take a look again.

Reply:
Looks great!